src/main/kotlin/lotto/LottoApplication.kt

```diff
@@ -0,0 +1,20 @@
+package lotto
+
+import lotto.domain.LottoMatcher
+import lotto.domain.LottoShop
+import lotto.domain.WinningNumber
+import lotto.view.InputView
+import lotto.view.ResultView
+
+fun main() {
+    val paidMoney = InputView.getPurchaseAmount()
+    val lottoTickets = LottoShop().buyLotto(paidMoney)
+    ResultView.showLottoInfo(lottoTickets)
+
+    val winningNumbers = InputView.getWinningNumbers()
+    LottoMatcher().apply {
+        val matchResult = matchResult(lottoTickets, WinningNumber(winningNumbers))
+        val earnedRate = calculateEarnedRate(matchResult.earnedMoney, paidMoney)
+        ResultView.showMatchResult(matchResult, earnedRate)
```

Comment:
개인적으로 `matchResult`나 `calculateEarnedRate`의 메서드가 어디에 있는 메서드인지 파악하기가 어렵습니다.
그래서 저는 1~2줄 코드 또는 프로퍼티만을 사용하는 코드가 아니면 다음과 같이 하는게 더 깔끔해 보이긴 합니다
```suggestion
    LottoMatcher().also { matcher ->
        val matchResult = matcher.matchResult(lottoTickets, WinningNumber(winningNumbers))
        val earnedRate = matcher.calculateEarnedRate(matchResult.earnedMoney, paidMoney)
        ResultView.showMatchResult(matchResult, earnedRate)
```